<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TabWidget</class>
 <widget class="QTabWidget" name="TabWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>308</width>
    <height>309</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Minimum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>308</width>
    <height>0</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>308</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Target Data</string>
  </property>
  <property name="currentIndex">
   <number>0</number>
  </property>
  <widget class="QWidget" name="tabCurrent">
   <attribute name="title">
    <string>Current</string>
   </attribute>
   <widget class="QLabel" name="labelTotal">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>10</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Total Records:</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelCurrent">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>51</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color:rgb(170, 0, 0);</string>
    </property>
    <property name="text">
     <string>Record # </string>
    </property>
    <property name="textFormat">
     <enum>Qt::AutoText</enum>
    </property>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>281</width>
      <height>31</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonClose">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>250</y>
      <width>61</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Close</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="currentGroup">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>281</width>
      <height>191</height>
     </rect>
    </property>
    <property name="title">
     <string>Current Record</string>
    </property>
    <widget class="QPushButton" name="buttonInsert">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>160</y>
       <width>61</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Insert</string>
     </property>
    </widget>
    <widget class="QPushButton" name="buttonRemove">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>160</y>
       <width>61</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Remove</string>
     </property>
    </widget>
    <widget class="QPlainTextEdit" name="textCurrent">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>261</width>
       <height>131</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="buttonPrev">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>250</y>
      <width>61</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Prev</string>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonNext">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>250</y>
      <width>61</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Next</string>
    </property>
   </widget>
   <zorder>currentGroup</zorder>
   <zorder>labelTotal</zorder>
   <zorder>labelCurrent</zorder>
   <zorder>line</zorder>
   <zorder>buttonClose</zorder>
   <zorder>buttonPrev</zorder>
   <zorder>buttonNext</zorder>
  </widget>
  <widget class="QWidget" name="tabAll">
   <attribute name="title">
    <string>All Data</string>
   </attribute>
   <widget class="QListView" name="listData">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>281</width>
      <height>231</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonClose_2">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>250</y>
      <width>61</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Close</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
